File: SofiForce.Web/ClientApp/src/app/core/Models/SearchModels/AppUserSearchModel.ts
import { BaseSearchModel } from "./baseSearchModel";

export interface AppUserSearchModel extends BaseSearchModel {
    appRoleId: number | null;
    userGroupId: number | null;
    isOnline: number | null;
    isLocked: number | null;
    lastLogin: Date | null;
    mustChangePassword: number | null;
}

File: SofiForce.Models/Models/SearchModels/AppUserSearchModel.cs
using System;
using System.Collections.Generic;

namespace Models
{
    public class AppUserSearchModel : BaseSearchModel
    {


		public Int32? AppRoleId;
		public Int32? UserGroupId;
		public int? IsOnline;
		public int? IsLocked;
		public DateTime? LastLogin;
		public int? MustChangePassword;
		
	}
}